docs/index.md

```diff
@@ -1,23 +1,23 @@
 # On becoming an OBO Semantic Engineer
 
-Welcome the OBOOK, the OBO Organized Knowledge portal and our OBO Semantic Engineering Training!
+Welcome to the OBOOK and our OBO Semantic Engineering Training!
 
-## Introduction to the OBOOK 
+## Introduction to the OBOOK (Open Biological and Biomedical Ontologies Organized Knowledge)
 
-Documentation in the OBOOK is organised based on the [Diataxis framework](https://diataxis.fr/) of documentations into 4 distinction sections: 
+Documentation in the OBOOK is organised into 4 distinct sections based on the [Diátaxis framework](https://diataxis.fr/) of documentation: 
 
-1. _Tutorials_: Tutorials are learning-oriented documentation in which there are exercises to help a beginner achieve basic competence in that area. 
-1. _How-to Guides_: How-to Guides are task-oriented documentation that function as directions that guide the reader through the steps to achieve a specific end.
-1. _Reference_: References are information-oriented documentation that describe, as succinctly as possible, and in an orderly way, a single topic.
-1. _Explanations_: Explanations are understanding-oriented documentation that clarifies, deepens and broadens the reader’s understanding of a subject.
+1. **Tutorials**: Learning-oriented documentation that contains exercises to help a beginner achieve basic competence in a specific area. 
```

Comment:
lol had to google if documentation was plural in itself >.< but yeah not 100% sure if documentation is plural in itself, does it mean that we use contains rather than contain? anyway super pedantic and I personally 100% do not notice stuff like that

Reply:
Thanks for checking! While not a definitive source, [this link](https://www.answers.com/Q/Is_documentation_singular_or_plural) describes "documentation" as singular (an uncountable mass noun).